README.md

```diff
@@ -44,6 +45,31 @@ The icon's [name](http://fontawesome.io/icons/), such as `fa-spinner` or `fa-squ
 <i class="fa fa-github"></i>
 ```
 
+##### alt
+For accessibility support, you can now add an *alt* attribute, which will implement a [screen-reader friendly](https://github.com/FortAwesome/Font-Awesome/issues/6133#issuecomment-88944728) wrapper for the icon.
+```html
+<fa name="github" alt="github website"></fa>
+<!-- rendered -->
+<span aria-hidden="true">
+      <i class="fa fa-github"></i>
+</span>
```

Comment:
For consistency's sake in terms of DOM nesting, I prefer the `aria-hidden` go directly on the `<i>`. Is there a case against that?